cmd/ast/main.go

```diff
@@ -98,10 +100,25 @@ func run(cmd *cobra.Command, args []string) error {
 		}
 		enc := json.NewEncoder(os.Stdout)
 		enc.SetIndent("", "  ")
-		caps, err := javascript.Language.CapabilityFinder.FindAllCapabilities(jsFile)
+		annots, err := javascript.Language.CapabilityFinder.FindAllCapabilities(jsFile)
 		if err != nil {
 			return err
 		}
+		var caps []core.Annotation
+		for _, v := range annots {
+			caps = append(caps, *v)
+		}
+		sort.Slice(caps, func(i, j int) bool {
```

Comment:
At least for me, it took a bit to realize what this is doing (just sorting the annotations by the order in which they appear in the code, right?). A comment might be nice:

```suggestion
		// sort the annotations by the order in which they appear in the source
		sort.Slice(caps, func(i, j int) bool {
```